src/components/Compendia/DownloadBlockForm.js

```diff
@@ -181,9 +168,9 @@ export const Download = ({ compendia }) => {
                   <ListItem
                     key={option.primary_organism_name}
                     label={formatString(option.primary_organism_name)}
-                    selectedOrganism={
-                      selectedOrganism
-                        ? formatString(selectedOrganism.primary_organism_name)
+                    compendium={
+                      compendium
+                        ? formatString(compendium.primary_organism_name)
```

Comment:
```suggestion
                    selected={compendium === option}
```

We should be able to just compare the references to the objects here.